### ISSUE TITLE
Laravel 5.6 not supported

### ISSUE THREAD

**Issue by Nowi5** [GrafiteInc/CMS#122]
Hi,

as you use  "illuminate/support": "5.*" this package is not ready for Laravel 5.6. Very sad.

**Comment by elanclarkson** [comment]
+1 I require Laravel 5.6 also. Looks like a number of dependencies need updating for this to work.

**Comment by boscowitch** [comment]
I did it manually and I had to doo basically 3 things to get it working and integrate it into an existing laravel 5.6 app.

1. update composer requirement 
 allow 5.6 (found this in a path commit of a fork)
` "illuminate/support": "5.5.*|5.6.*",`

2. upgrade the markdown package in the composer file  since it blocked newer laravel packages too:

```
-        "graham-campbell/markdown": "^9.0",
+        "graham-campbell/markdown": "^10.0",
```


And lastly not really on the quarx side;

3. Add a npm package bootstrap-sass into the existing laravel app to make npm run dev work and have working stylesheets in the browser.
```
    "dependencies": {
        "bootstrap-sass": "^3.3.7"
    }
```


So far everything seems to work (just logged in ect. and clicked through the dasboard) 
I will gradually test more and more  functions as I go... 

EDIT:
Tested so far:
images upload  (I just first forgot the php artisan storage:link for it to be viewed but works )
creating blog article with images
create event
widgets and  add widget to the default theme and edit widget from the frontend button
so far everything seems to work as expected
